README.md

```diff
@@ -184,6 +184,14 @@ The following tests run on a Linux DSVM daily.
 | **Windows Spark** | main | [![Build Status](https://dev.azure.com/best-practices/recommenders/_apis/build/status/windows-tests/dsvm_nightly_win_pyspark?branchName=main)](https://dev.azure.com/best-practices/recommenders/_build/latest?definitionId=103&branchName=main) | | staging | [![Build Status](https://dev.azure.com/best-practices/recommenders/_apis/build/status/windows-tests/dsvm_nightly_win_pyspark?branchName=staging)](https://dev.azure.com/best-practices/recommenders/_build/latest?definitionId=103&branchName=staging) |
 -->
 
+### AzureML Nightly Build Status
+
+Smoke and integration tests are run daily on AzureML.
+
+| Build Type | Branch | Status |  | Branch | Status |
+| --- | --- | --- | --- | --- | --- |
+| **Nightly** | main | [![azureml-nightly](https://github.com/microsoft/recommenders/actions/workflows/azureml-nightly.yml/badge.svg?branch=main)](https://github.com/microsoft/recommenders/actions/workflows/azureml-nightly.yml?query=branch%3Amain) | | staging | [![azureml-nightly](https://github.com/microsoft/recommenders/actions/workflows/azureml-nightly.yml/badge.svg?branch=staging)](https://github.com/microsoft/recommenders/actions/workflows/azureml-nightly.yml?query=branch%3Astaging) |
```

Comment:
is it possible to get 3 badges, CPU, GPU and Spark?

Reply:
GitHub badges are workflow specific, not job specific. We could create separate workflows for CPU, GPU, Spark.
PS There is a somewhat convoluted way to get a job status in GitLab as mentioned here https://stackoverflow.com/a/47343010. In our case, for example, CPU status depends on multiple jobs (`group_cpu_001`, `group_cpu_002`, ...) within the workflow. If one of the groups fail, the badge will not be created.

Reply:
> If one of the groups fail, the badge will not be created.

when you say that is not created, what exactly happens? do you get a 404 or do you get a red badge? What we want is that if one of the groups fail, we get a red badge, if we click on the badge, then it takes you to the error. Do you think this is possible?

Reply:
Let's say we have the CPU badge generation step in the workflow that depends on three jobs, one for each group. If one of the jobs fails, the badge generation step will be cancelled and we will get 404.
I was not able to find a way to link the badge to the latest run/error. The best I could do here is link the badge to the Actions page with the corresponding branch filter applied.

Reply:
got it, feel free to merge whenever you want